katello/hooks/pre/10-reset_feature.rb

```diff
@@ -135,6 +136,11 @@ def reset_pulp
   )
 end
 
+def reset_pulcore
```

Comment:
Kind of petty but should that be `reset_pulpcore`?

Reply:
Updated.. :+1: